src/Page.js

```diff
@@ -198,6 +198,12 @@ class Page extends React.Component {
 		window.history.replaceState({ panel: homePanel }, homePanel);
 	}
 
+	UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
This method is UNSAFE

Reply:
Use [this](https://ru.reactjs.org/docs/react-component.html#static-getderivedstatefromprops) method